main.cpp

```diff
@@ -74,67 +101,123 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 /*
  1)
  */
+bool enableLight (bool switchState, int switchSize = 150, bool peopleHome = false)
+{
+    ignoreUnused(switchState, switchSize, peopleHome);
+    return {};
+}
 
 /*
  2)
  */
+void drawGraphics(bool windowFocus, float windowSize = 15.24f, int windowHeight = 123)
+{
+    ignoreUnused(windowFocus, windowSize, windowHeight);
+}
 
 /*
  3)
  */
+float getFrequency (float amplitude, float time)
+{
+    ignoreUnused(amplitude, time);
+    return {};
+}
 
 /*
  4)
  */
+int setColor (int red, int blue, int green = 254)
+{
+    ignoreUnused(red, blue, green);
+    return {};
+}
 
 /*
  5)
  */
+double increaseTemperature (int amount, bool raining)
+{
+    ignoreUnused (amount, raining);
+    return {};
+}
 
 /*
  6)
  */
+void displayCredits (float versionNumber, char initials = 'a')
+{
+    ignoreUnused(versionNumber, initials);
+}
 
 /*
  7)
  */
+char createRandomLetter (int randomSeed, char startingLetter)
+{
+    ignoreUnused(randomSeed, startingLetter);
+    return {};
+}
 
 /*
  8)
  */
+double setBPM (int tempo, int divisions = 4)
+{
+    ignoreUnused(tempo, divisions);
+    return {};
+}
 
 /*
  9)
  */
+void calibrateSpeakers (double xPosition, double yPosition, int loudness = 60)
+{
+    ignoreUnused(xPosition, yPosition, loudness);
+}
 
 /*
  10)
  */
+ void insertEffect (bool slotFree, int slotNumber)
+ {
+    ignoreUnused(slotFree, slotNumber);
+ }
 
 int main()
 {
     //example of calling that function
     rentACar(6, 2); 
     
     //1)
+    rentACar (250, 5);
     
-    //2)
+    //2) 
+    drawGraphics(true); 
     
     //3)
-    
+    getFrequency(0.5f, 125.5f);
+
     //4)
+    setColor (50, 40, 20);
     
     //5)
+    increaseTemperature(100, false);
     
     //6)
+    displayCredits(1.254f);
     
     //7)
+    createRandomLetter (125, 'f');
     
     //8)
+    setBPM (120);
     
     //9)
+    calibrateSpeakers(15.235, 5.524);
     
     //10)
+    insertEffect(true, 1);
```

Comment:
I would like you to store the returned value from any of these functions that return something into a local variable.  
i.e. `auto c = createRandomLetter(125, 'f');`
At the end of `main()`, pass all of them to the `ignoreUnused()` function like you did in `variableDeclarations()`, before the "good to go" line

- [ ] store returned values in local variables, pass them to `ignoreUnused()`